.github/ISSUE_TEMPLATE/support-request.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Support Request
+about: Request Support
+title: "[SUPPORT REQEUST] CLOSE"
+labels: wontfix
+assignees: ''
+
+---
+
+**ALL SUPPORT REQUESTS MUST BE MADE VIA DISCORD - CREATING A REQUEST HERE WILL RESULT IN IMMEDIATE CLOSURE OF THE REQUEST.**
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-emphasis-as-heading] Don’t use emphasis to introduce a section, use a heading](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5471038)